docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/templates/cards/node--embedded-feature-card.html.twig

```diff
@@ -17,5 +17,10 @@
   {% set image = content.field_image_article|field_value %}
 {% endif %}
 
+{% set id = node.id()|clean_unique_id %}
```

Comment:
Did you ever test if you can have multiple instances of the same card and that the IDs are unique?

Reply:
Derek did test.